js/build.js

```diff
@@ -14,3 +14,9 @@ Fliplet.Widget.instance('notification-inbox-1-0-0', function (data) {
     return Promise.reject();
   });
 });
+
+Fliplet.Analytics.trackEvent({
+  category: 'notification_inbox',
```

Comment:
can you check with Hugo that using underscores is the standard? just want to check for consistency before it's too late